Classes/Controller/Backend/Search/IndexQueueModuleController.php

```diff
@@ -147,21 +167,10 @@ public function initializeIndexQueueAction()
                 FlashMessage::OK
             );
 
-            $this->addFlashMessagesByQueueIdentifier('solr.queue.initializer');
+            //$this->addFlashMessagesByQueueIdentifier('solr.queue.initializer');
```

Comment:
I've tested the module with TYPO3 8.7 LTS. Looks good. One thing i think we should optimize is to keep the selected page/site when the module is changed. Is that possible?